C/Bankers Algorithm/bankersAlgorithm.c

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright [2020] [Abdul Majed]

```

Comment:
Remove the square brackets wrapping the year and your name.

`Copyright 2020 Abdul Majed`